README.md

```diff
@@ -96,19 +97,37 @@ The file `app.entitlements` must have following content:
 </plist>
 ```
 
-For further details have a look at [Apple's documentation](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_hypervisor).
+Please consult [Apple's Documentation](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_security_hypervisor) for more information.
 
 ## Running Hermit apps within Uhyve
 
-Use the hypervisor to start the unikernel.
+Assuming that you have **installed** Uhyve, run the hypervisor to start the unikernel:
 ```sh
 uhyve /path/to/the/unikernel/binary
 ```
 
+> [!NOTE]
+> This repository ships a few binaries that can be used for testing.
+>
+> If you want to compile Hermit binaries yourself (or create your own), take a look at the following repositories:
+> - [hermit-os/hermit-rs](https://github.com/hermit-os/hermit-rs)
+> - [hermit-os/hermit-rs-template](https://github.com/hermit-os/hermit-rs-template)
+> - [hermit-os/hermit-playground](https://github.com/hermit-os/hermit-playground)
+
+### Testing using `cargo run`
+
+If you are hacking on Uhyve's source code, you can also use `cargo run` to immediately test your changes:
+
+```sh
+cargo run -- -v data/x86_64/rusty_demo
+cargo run -- -v data/x86_64/hello_world
+cargo run -- -v data/x86_64/hello_c
+```
+
```

Comment:
This rather should be moved to some kind of Wiki or a later section in the document